src/components/FarmingList/ListItem.jsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import { ExternalLink } from 'react-feather';
+import { useMedia } from 'react-use';
+import { Flex } from 'rebass';
+import styled from 'styled-components';
+
+import { USD } from '@swapr/sdk';
+
+import { Typography } from '../../Theme';
+import carrotListLogoUrl from '../../assets/images/carrot.png';
+import { ChainId } from '../../constants';
+import { useNativeCurrencySymbol, useNativeCurrencyWrapper, useSelectedNetwork } from '../../contexts/Network';
+import { formatDollarAmount, formattedNum, getSwaprLink } from '../../utils';
+import DoubleTokenLogo from '../DoubleLogo';
+import FormattedName from '../FormattedName';
+import Link, { InternalListLink } from '../Link';
+import TokenLogo from '../TokenLogo';
+
+const carrotTokenRegex = new RegExp(/g([a-zA-z]*)-\d{4}$/);
+
+export const DashGrid = styled.div`
+  display: grid;
+  grid-gap: 1em;
+  grid-template-columns: 0.5fr 0.9fr 0.5fr 0.4fr 0.2fr;
+  grid-template-areas: 'pair tvl apy rewardTokens link';
+  padding: 10px 36px;
+  height: 100px;
+
+  > * {
+    justify-content: flex-end;
+
+    :first-child {
+      justify-content: flex-start;
+      text-align: left;
+    }
+  }
+
+  @media screen and (min-width: 680px) {
+    display: grid;
+    grid-gap: 1em;
+    grid-template-columns: 0.6fr 0.6fr 0.6fr 0.6fr 0.4fr 0.2fr;
+    grid-template-areas: 'pair tvl yield apy rewardTokens link';
+
+    > * {
+      justify-content: flex-end;
+      width: 100%;
+
+      &:first-child {
+        justify-content: flex-start;
+      }
+    }
+  }
+
+  @media screen and (min-width: 1081px) {
+    display: grid;
+    grid-gap: 0.5em;
+    grid-template-columns: 0.2fr 1fr 1fr 1fr 1fr 1fr 0.5fr;
+    grid-template-areas: 'index pair tvl yield apy rewardTokens link';
+  }
+`;
+
+const FlexText = ({ area, justifyContent, flexDirection, color, children }) => (
+  <Flex area={area} justifyContent={justifyContent} flexDirection={flexDirection || 'row'}>
+    <Typography.LargeText color={color || 'text1'} sx={{ display: 'flex', alignItems: 'center' }}>
+      {children}
+    </Typography.LargeText>
+  </Flex>
+);
+
+export default function ListItem({ campaign, index, nativeCurrencyPrice }) {
+  const below740 = useMedia('(max-width: 740px)');
+  const below600 = useMedia('(max-width: 600px)');
+  const below680 = useMedia('(max-width: 680px)');
+  const below1080 = useMedia('(max-width: 1080px)');
+
+  const nativeCurrency = useNativeCurrencySymbol();
+  const nativeCurrencyWrapper = useNativeCurrencyWrapper();
+  const selectedNetwork = useSelectedNetwork();
+
+  const carrotLogo = `${window.location.origin}${
+    carrotListLogoUrl.startsWith('.') ? carrotListLogoUrl.substring(1) : carrotListLogoUrl
+  }`;
```

Comment:
Is it required?

Reply:
Yes, we can use `carrotListLogoUrl` directly